R/uvhydrograph-data.R

```diff
@@ -44,24 +44,34 @@ parseUVData <- function(data, plotName, month) {
   
   if(plotName == "secondary"){
     
-    corr_UV2 <- subsetByMonth(getTimeSeries(data, "downsampledSecondarySeries"), month)
-    est_UV2 <- subsetByMonth(getTimeSeries(data, "downsampledSecondarySeries", estimatedOnly=TRUE), month)
-    uncorr_UV2 <- subsetByMonth(getTimeSeries(data, "downsampledSecondarySeriesRaw"), month)
-    
-    series_corr2 <- subsetByMonth(getCorrections(data, "secondarySeriesCorrections"), month)
+    #Reference Time Series Data
+    corr_UV_ref <- subsetByMonth(getTimeSeries(data, "downsampledReferenceSeries"), month)
+    est_UV_ref <- subsetByMonth(getTimeSeries(data, "downsampledReferenceSeries", estimatedOnly=TRUE), month)
+    uncorr_UV_ref <- subsetByMonth(getTimeSeries(data, "downsampledReferenceSeriesRaw"), month)
+    series_corr_ref <- subsetByMonth(getCorrections(data, "referenceSeriesCorrections"), month)
+    approvals_ref <- getApprovals(data, chain_nm="downsampledReferenceSeries", legend_nm=getTimeSeriesLabel(data, "downsampledReferenceSeries"),
+                                  appr_var_all=c("appr_approved", "appr_inreview", "appr_working"),
+                                  subsetByMonth=TRUE, month=month, point_type=73)
+
+    #Upchain Time Series Data
+    corr_UV_up <- subsetByMonth(getTimeSeries(data, "downsampledUpchainSeries"), month)
+    est_UV_up <- subsetByMonth(getTimeSeries(data, "downsampledUpchainSeries", estimatedOnly=TRUE), month)
+    uncorr_UV_up <- subsetByMonth(getTimeSeries(data, "downsampledUpchainSeriesRaw"), month)
+    series_corr_up <- subsetByMonth(getCorrections(data, "upchainSeriesCorrections"), month)
+    approvals_up <- getApprovals(data, chain_nm="downsampledUpchainSeries", legend_nm=getTimeSeriesLabel(data, "downsampledUpchainSeries"),
+                                 appr_var_all=c("appr_approved", "appr_inreview", "appr_working"),
+                                 subsetByMonth=TRUE, month=month, point_type=73)
     
     effect_shift <- subsetByMonth(getTimeSeries(data, "effectiveShifts"), month)
     gage_height <- subsetByMonth(getMeanGageHeights(data), month)
     gw_level <- subsetByMonth(getGroundWaterLevels(data), month)
     meas_shift <- subsetByMonth(getFieldVisitMeasurementsShifts(data), month)
+    approvals <- approvals_ref
+    approvals <- append(approvals, approvals_up)
```

Comment:
You can just do this in one step: `approvals <- append(approvals_ref, approvals_up)`